script.js

```diff
@@ -1,76 +1,121 @@
 //fixed variables
 CHOOSENUM = "Choose Number";
 ACTUALGAME = "Actual Game";
+NORMAL = "normal";
+LOWESTWINS = "lowest wins";
 
+//variables
 var player1num = 0;
 var player2num = 0;
+var player1sum = 0;
+var player2sum = 0;
 var player = 1;
 var section = CHOOSENUM;
+var gameMode = "";
 var roll1 = 0;
 var roll2 = 0;
 
-//variables
+//message
+var chooseOrderMessage = function (player, roll1, roll2) {
+  return `Player ${player}, please choose the order which order you want. Either <br><br> 1: ${roll1} before ${roll2} <br> 2: ${roll2} before ${roll1}`;
+};
+
 var main = function (input) {
+  if (!gameMode) {
+    if (!input) {
+      return "Please input: 'normal' or 'lowest wins'";
+    }
+
+    if (!(input == NORMAL || input == LOWESTWINS)) {
+      return "Incorrect input. <br> Please input: 'normal' or 'lowest wins'";
+    }
+
+    gameMode = input;
+    return " Welcome players, press submit start the game";
+  }
+
   //input for player 1
   if (section == CHOOSENUM) {
+    if (roll1 == 0) {
+      roll1 = randomNumber();
+      roll2 = randomNumber();
+    }
+
     if (player == 1) {
       if (!input) {
-        roll1 = randomNumber();
-        console.log("Player 1", roll1);
-        roll2 = randomNumber();
-        console.log("Player 1", roll2);
-        return `Player ${player}, please choose the order which order you want. Either <br><br> 1: ${roll1} before ${roll2} <br> 2: ${roll2} before ${roll1}`;
+        return chooseOrderMessage(player, roll1, roll2);
       }
 
       if (!(input == 1 || input == 2)) {
-        return `Incorrect input!! <br> Player ${player}, please choose the order which order you want. Either <br><br> 1: ${roll1} before ${roll2} <br> 2: ${roll2} before ${roll1}`;
+        return (
+          "Incorrect input!! <br>" + chooseOrderMessage(player, roll1, roll2)
+        );
       }
 
       var playerChoice = input;
```

Comment:
[Suggestion] Is this really necessary? Since you are just using input, you can just leave it as input.